src/test/java/com/mavenit/selenium/training/page_elements/HomePage.java

```diff
@@ -0,0 +1,55 @@
+package com.mavenit.selenium.training.page_elements;
+
+import com.mavenit.selenium.training.Hooks;
+import com.mavenit.selenium.training.utils.RandomNumberHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static junit.framework.TestCase.fail;
+
+public class HomePage extends Hooks {
+
+    public static String searchitem;
+
+    public void search(String item) {
+        driver.findElement(By.id("search")).clear();
+        driver.findElement(By.id("search")).clear();
+
+        searchitem = item;
+        enterSearchItem(item);
+        clickSearch();
+    }
+
+    public void enterSearchItem(String item) {
+        driver.findElement(By.id("search")).sendKeys(item);
+    }
+
+    private void clickSearch() {
+        driver.findElement(By.cssSelector(".dc-search-fieldset__submit-button")).click();
+
+    }
+
+    public void selectDepartmentFromSuggestions(String item) {
+        selectFromSuggestions(item, By.cssSelector(".suggestion dc-search-suggestions__suggestion--term"));
+    }
```

Comment:
hey above locator is invalid
right locator  "dc-search-suggestions__suggestion--term"

Reply:
thanks updated